after/syntax/perl.vim

```diff
@@ -0,0 +1,38 @@
+if dracula#should_abort('perl')
+    finish
+endif
+
+" Regex
+hi! link perlMatchStartEnd       DraculaRed
+
+" Builtin functions
+hi! link perlOperator            DraculaCyan
+hi! link perlStatementFiledesc   DraculaCyan
+hi! link perlStatementFiles      DraculaCyan
+hi! link perlStatementFlow       DraculaCyan
+hi! link perlStatementHash       DraculaCyan
+hi! link perlStatementIOfunc     DraculaCyan
+hi! link perlStatementIPC        DraculaCyan
+hi! link perlStatementList       DraculaCyan
+hi! link perlStatementMisc       DraculaCyan
+hi! link perlStatementNetwork    DraculaCyan
+hi! link perlStatementNumeric    DraculaCyan
+hi! link perlStatementProc       DraculaCyan
+hi! link perlStatementPword      DraculaCyan
+hi! link perlStatementRegexp     DraculaCyan
+hi! link perlStatementScalar     DraculaCyan
+hi! link perlStatementSocket     DraculaCyan
+hi! link perlStatementTime       DraculaCyan
+hi! link perlStatementVector     DraculaCyan
+
+" Highlighting for quoting constructs, tied to existing option in vim-perl
+if get(g:, 'perl_string_as_statement', 0)
+  hi! link perlStringStartEnd DraculaRed
```

Comment:
Is there a need for the conditional here?

Reply:
@dsifford not an answer to your question, but cf. `:help ft-perl-syntax`

Reply:
I didn't want to override it completely (appears yellow by default, same as the string itself), and when the option is enabled it changes to pink, which is the same color as escaped characters within strings. I decided that changing the color when the option is enabled would be the best option.

Reply:
Got it. Sounds good.